packages/commonwealth/client/scripts/views/components/user/user.tsx

```diff
@@ -219,14 +226,14 @@ export const User = (props: UserAttrs) => {
             )
           ) : (
             <a className="user-display-name username">
-              {!profile || !profile?.id ? (
-                !profile?.id ? (
-                  `${profile.address.slice(0, 8)}...${profile.address.slice(
-                    -5
-                  )}`
+              {!profile ? (
+                showAsDeleted ? (
+                  'Deleted'
                 ) : (
-                  addrShort
+                  'Anonymous'
```

Comment:
I think we want to show the addrShort UNLESS the address meets the conditions where we were showing AnonymousUsers, i.e.:
```
      comment.authorChain !== app.chain.id &&
      comment.authorChain !== app.chain.base
```

Reply:
Yeah that is what's happening here; the entire ternary changed because it was written in a weird way so this isnt actually just swapping out "addrShort" for "Anonymous".